src/features/web-compat.js

```diff
@@ -51,8 +51,12 @@ export default class WebCompat extends ContentFeature {
         Notification.maxActions = 2
 
         // Expose the API
-        // @ts-expect-error window.Notification isn't assignable
-        window.Notification = Notification
+        this.defineProperty(window, 'Notification', {
+            value: Notification,
+            writable: true,
+            configurable: true,
+            enumerable: false
+        })
```

Comment:
You could use `wrapProperty()` here to get runtime shape checks

Reply:
No we can't it doesn't exist: https://github.com/duckduckgo/content-scope-scripts/blob/11e62c5311f705ad0b461184a614daaaf5f9f32a/src/content-feature.js#L320-L322

Reply:
ah, right, sorry